locales/fr.po

```diff
@@ -1764,7 +1764,7 @@ msgstr "Précisément, quand vous intégrez des questions ou des tableaux de bor
 msgid "Enable"
 msgstr "Activer"
 
-#. Que pensez-vous de "intégrer" / "intégration" / etc ?
+#. Que pensez-vous de "intégrer" / "intégration" / etc ?
```

Comment:
This was automatically done by my editor trimming whitespace from the end of the file. It's just a comment, so it shouldn't matter, but to be sure...

With whitespace highlighting on:
```
git config diff.wsErrorHighlight all
```
the git diff is:
```diff
diff --git a/locales/fr.po b/locales/fr.po
index 9142c585f8..fc28db5d69 100644
--- a/locales/fr.po
+++ b/locales/fr.po
@@ -1764,7 +1764,7 @@ msgstr "Précisément, quand vous intégrez des questions ou des tableaux de bor
 msgid "Enable"
 msgstr "Activer"
 
-#. Que pensez-vous de "intégrer" / "intégration" / etc ?^M
+#. Que pensez-vous de "intégrer" / "intégration" / etc ?
 #. (pour "embed", "embedding", etc)
 #: frontend/src/metabase/admin/settings/components/widgets/EmbeddingLevel.jsx:28
 msgid "Premium embedding enabled"
@@ -30398,7 +30398,7 @@ msgstr "Fuseau horaire incorrect {0} : {1}"
 
...
```

`^M` indicates the carriage return character. It was written with a windows computer.

Reply:
Interesting that there is only one such line in the whole file.